.github/scale-config.yml

```diff
@@ -18,6 +18,12 @@
 #     is_ephemeral: true
 
 runner_types:
+  # mainly used for ciflow-should-run, not made to run any serious tests
+  linux.large:
+    instance_type: c5.large
+    os: linux
+    disk_size: 10
```

Comment:
Is 10 Gigs enough ? Maybe we should bump up space to something like 40Gig so we don't need to worry about this ?

Reply:
It's only use is on a no-op job so we shouldn't actually be using it for anything other than `ciflow-should-run`